pilot/cmd/istioctl/main.go

```diff
@@ -90,6 +90,7 @@ and destination policies.
 
 `,
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			_ = log.Configure(loggingOptions)
```

Comment:
Can you switch to PersistentPreRunE and return on error? 

I'm not sure why this would fail in practice, but dropping logs isn't good.

Reply:
+1 ... cool.. didn't realize there was a PreRunE.  Will fix